templates/html/templates/common/macros.html

```diff
@@ -99,9 +109,9 @@
 {%- endmacro %}
 
 
-{% macro render_description_item_local_iri(result_set, property_id, property_label, link_label_property_id=None, accordion_style=False) -%}
+{% macro render_description_item_iri(result_set, property_id, property_label, rdfa_property, link_label_property_id=None, accordion_style=False, accordion_content_state="hide", external_link=False) -%}
```

Comment:
same as above: rename the paramaters `accordion_style`, `accordion_content_state`, and `external_link`